### ISSUE TITLE
Source (reference) of the implementation

### ISSUE THREAD

**Issue by akmandor** [impulsecorp/HypRL#1]
Is your implementation based on [this paper](https://arxiv.org/abs/1906.11527)?

**Comment by peter-ch** [comment]
No. This code attempts to construct neural networks by selecting and adjusting the NN weights as if it was a computer game, also adding/removing nodes. It uses RL to learn how to do that, but it doesn't do hyperparameter optimization.

**Comment by NimmiW** [comment]
@akmandor https://github.com/hadijomaa/HypRL
